<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>885</width>
    <height>620</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>PiTres - Pirate Treasurer Helper Application</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../PiTres.qrc">
    <normaloff>:/resources/application_logo.svg</normaloff>:/resources/application_logo.svg</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QGridLayout" name="gridLayout"/>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>885</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="separator"/>
    <addaction name="action_Exit"/>
   </widget>
   <widget class="QMenu" name="menu_Import">
    <property name="title">
     <string>Functions</string>
    </property>
    <addaction name="actionFrom_LDAP"/>
    <addaction name="actionInvoiceWizard"/>
    <addaction name="actionAdjustDates"/>
    <addaction name="actionDonation"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionConfiguration"/>
    <addaction name="separator"/>
    <addaction name="actionHelp"/>
    <addaction name="action_About_PiTres"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Import"/>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionFrom_LDAP"/>
   <addaction name="actionImport_Payments"/>
   <addaction name="actionAdjustDates"/>
   <addaction name="actionInvoiceWizard"/>
   <addaction name="actionDonation"/>
  </widget>
  <widget class="QDockWidget" name="dockWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="features">
    <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable</set>
   </property>
   <property name="windowTitle">
    <string>Views</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QToolButton" name="userlistButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>Show all Users, only for your current Section or for all if you are allowed so</string>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Show Users</string>
       </property>
       <property name="icon">
        <iconset resource="../PiTres.qrc">
         <normaloff>:/icons/resources/icons/businessman.png</normaloff>:/icons/resources/icons/businessman.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>64</width>
         <height>64</height>
        </size>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextUnderIcon</enum>
       </property>
       <property name="arrowType">
        <enum>Qt::NoArrow</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="sentBillsButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>Show all sent Bills and Reminds and send new Reminds as well</string>
       </property>
       <property name="text">
        <string>Show sent Bills</string>
       </property>
       <property name="icon">
        <iconset resource="../PiTres.qrc">
         <normaloff>:/icons/resources/icons/money.png</normaloff>:/icons/resources/icons/money.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>64</width>
         <height>64</height>
        </size>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextUnderIcon</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="importPaymentsButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>Create new Bills and send they as EMails or print out the PDFs and Letters</string>
       </property>
       <property name="text">
        <string>Donations and Dues</string>
       </property>
       <property name="icon">
        <iconset resource="../PiTres.qrc">
         <normaloff>:/icons/resources/icons/chest.png</normaloff>:/icons/resources/icons/chest.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>64</width>
         <height>64</height>
        </size>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextUnderIcon</enum>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="action_Exit">
   <property name="icon">
    <iconset resource="../PiTres.qrc">
     <normaloff>:/icons/resources/icons/application-exit.png</normaloff>:/icons/resources/icons/application-exit.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionFrom_LDAP">
   <property name="icon">
    <iconset resource="../PiTres.qrc">
     <normaloff>:/icons/resources/icons/data_refresh.png</normaloff>:/icons/resources/icons/data_refresh.png</iconset>
   </property>
   <property name="text">
    <string>Import from &amp;LDAP</string>
   </property>
   <property name="toolTip">
    <string>Synchronize the USertable with the LDAP-Database</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="actionInvoiceWizard">
   <property name="icon">
    <iconset resource="../PiTres.qrc">
     <normaloff>:/icons/resources/icons/cashier.png</normaloff>:/icons/resources/icons/cashier.png</iconset>
   </property>
   <property name="text">
    <string>Invoice Wizard</string>
   </property>
   <property name="toolTip">
    <string>Create new Invoices based on different Selections and Settings</string>
   </property>
  </action>
  <action name="action_About_PiTres">
   <property name="icon">
    <iconset resource="../PiTres.qrc">
     <normaloff>:/resources/application_logo.svg</normaloff>:/resources/application_logo.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;About PiTres</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="icon">
    <iconset resource="../PiTres.qrc">
     <normaloff>:/icons/resources/icons/system-help.png</normaloff>:/icons/resources/icons/system-help.png</iconset>
   </property>
   <property name="text">
    <string>Help</string>
   </property>
  </action>
  <action name="actionShow_Users">
   <property name="icon">
    <iconset>
     <normaloff>:/buttons/resources/buttons/user-identity.png</normaloff>:/buttons/resources/buttons/user-identity.png</iconset>
   </property>
   <property name="text">
    <string>Show &amp;Users</string>
   </property>
   <property name="toolTip">
    <string>Show all USers, either from you Section or from all if you are allowed</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+U</string>
   </property>
  </action>
  <action name="actionShow_Bills">
   <property name="icon">
    <iconset>
     <normaloff>:/buttons/resources/buttons/mail-folder-outbox.png</normaloff>:/buttons/resources/buttons/mail-folder-outbox.png</iconset>
   </property>
   <property name="text">
    <string>Show Bills</string>
   </property>
   <property name="toolTip">
    <string>Show all sent Bills and Reminds and send new Reminds</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionImport_Payments">
   <property name="icon">
    <iconset resource="../PiTres.qrc">
     <normaloff>:/icons/resources/icons/payment_in.png</normaloff>:/icons/resources/icons/payment_in.png</iconset>
   </property>
   <property name="text">
    <string>Import Payments</string>
   </property>
   <property name="toolTip">
    <string>Import Payments based on the Reference-Number, the Member UID, the Member name or based on ESR</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+B</string>
   </property>
  </action>
  <action name="actionConfiguration">
   <property name="icon">
    <iconset resource="../PiTres.qrc">
     <normaloff>:/icons/resources/icons/preferences-system.png</normaloff>:/icons/resources/icons/preferences-system.png</iconset>
   </property>
   <property name="text">
    <string>Configuration</string>
   </property>
   <property name="toolTip">
    <string>Configure PiTres</string>
   </property>
  </action>
  <action name="actionAdjustDates">
   <property name="icon">
    <iconset resource="../PiTres.qrc">
     <normaloff>:/icons/resources/icons/adjust_members.png</normaloff>:/icons/resources/icons/adjust_members.png</iconset>
   </property>
   <property name="text">
    <string>Adjust Member Dates</string>
   </property>
   <property name="toolTip">
    <string>Adjust Member Dates</string>
   </property>
  </action>
  <action name="actionDonation">
   <property name="icon">
    <iconset resource="../PiTres.qrc">
     <normaloff>:/icons/resources/icons/chest.png</normaloff>:/icons/resources/icons/chest.png</iconset>
   </property>
   <property name="text">
    <string>Show Donation</string>
   </property>
   <property name="toolTip">
    <string>Show Donaitons and Section Dues</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../PiTres.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>action_Exit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>importPaymentsButton</sender>
   <signal>clicked()</signal>
   <receiver>actionImport_Payments</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>66</x>
     <y>310</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sentBillsButton</sender>
   <signal>clicked()</signal>
   <receiver>actionShow_Bills</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>66</x>
     <y>220</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>userlistButton</sender>
   <signal>clicked()</signal>
   <receiver>actionShow_Users</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>66</x>
     <y>130</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
